data/net/minecraft/server/players/StoredUserList.mapping

```diff
@@ -13,5 +15,7 @@ CLASS net/minecraft/server/players/StoredUserList
 		ARG 1 obj
 	METHOD remove (Ljava/lang/Object;)V
 		ARG 1 entry
+	METHOD remove (Lnet/minecraft/server/players/StoredUserEntry;)V
+		ARG 1 user
```

Comment:
The naming of the parameters of these twomethods should be interchanged; the first one removes a user, while the second one accepts an entry and removes the user for that entry.